hexane/ibi_all/grompp.mdp

```diff
@@ -50,6 +50,9 @@ ns_type                  = grid
 pbc                      = xyz
 ; nblist cut-off        
 rlist                    = 1.5
+;cutoff-scheme
+cutoff-scheme            = group
+
```

Comment:
Drop this block, it is automatically done in `run_gromacs.sh`. (Background: gromacs-2017 won't have group kernels anymore.)